perspective_reddit_bot/check_mod_actions.py

```diff
@@ -78,6 +98,17 @@ def _main():
   parser.add_argument('output_path', help='path to write output file')
   parser.add_argument('-id_key', help='json key containing reddit comment id',
                       default='comment_id')
+  parser.add_argument('-mod_creds',
+                      help=('whether the bot has mod credentials. if set,'
+                            ' the output contains "approved", "removed", and'
+                            ' "deleted" fields.'),
+                      dest='has_mod_creds',
+                      action='store_true')
+  parser.add_argument('-no_mod_creds',
+                      help=('the bot does not have mod credentials. the output'
+                            ' only contains "removed" and "deleted" fields,'),
+                      dest='has_mod_creds',
+                      action='store_false')
```

Comment:
Is there a reason to have "no_mod_creds" as a flag?  Shouldn't `args.mod_creds == False` mean the same thing?

Reply:
It's a little hard to have a boolean flag in argparse: https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse

I want the user to explicitly specify whether they have mod creds, and this is the best I could do.